### PR TITLE
SDO-513: Add security recommendations on manual install

### PR DESCRIPTION
**Pull request by mota-lhd**
**Description**

docs: add security recommendations page

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by Erwan-loot — APPROVED** [review]

**Review by Erwan-loot — CHANGES_REQUESTED** [review]
It should be pointed out that you need to ask support if customers want ip authorization. 
It would be necessary to delete the mentions of the codes with the if

File: install_pim/manual/security_recommendations.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		{% if adminer_ips_allowed is defined %}
+		Require ip {{ adminer_ips_allowed }}
+		{% endif %}
+		Require all denied
+	</Location>
+
+	<Location /app_dev.php>
+		{% if profiler_ips_allowed is defined %}
+		Require ip {{ profiler_ips_allowed }}
+		{% endif %}
```
Comment (Erwan-loot) [inline install_pim/manual/security_recommendations.rst:32]:
```suggestion
	<Location /adminer.php>

		Require all denied
	</Location>

	<Location /app_dev.php>

```

File: install_pim/manual/security_recommendations.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		Require all denied
+	</Location>
```
Comment (Erwan-loot) [inline install_pim/manual/security_recommendations.rst:34]:
```suggestion
	<Location /adminer.php>
		Require all denied
	</Location>

	<Location /app_dev.php>
		Require all denied
	</Location>
```

File: install_pim/manual/security_recommendations.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Rather than directly attacking a server, attackers will first run discovery tasks to steal publicly accessible data off the server.
+
+There are many types of sensitive information that you should protect from attackers, including system data, configuration, secrets, 
+intellectual property and an individual's personal (private) information.
+
+You should never enable these in production environments as it will lead to major security vulnerabilities in your project.
+- `adminer <https://www.adminer.org/>`_
+- `Php Symfony Profiler <https://symfony.com/doc/current/profiler.html>`_
+
+On test/dev/staging/pre-prod environments, if needed only, make sure to enable these with IP filtering like the following configuration
```
Comment (Erwan-loot) [inline install_pim/manual/security_recommendations.rst:16]:
```suggestion
On test/dev/staging/pre-prod environments, if needed only, make sure to ask support team to add with authorized IPs
```

**Review by Erwan-loot — CHANGES_REQUESTED** [review]

**Review by Erwan-loot — APPROVED** [review]

**Review by AdeOlubi — APPROVED** [review]

**Review by niarcotyk — APPROVED** [review]

**Review by Airmanbzh — APPROVED** [review]

**Pull request merged**